src/pages/registration/contributors_tab/Authors.tsx

```diff
@@ -32,6 +33,13 @@ const Authors = ({ push, replace }: AuthorsProps) => {
   const [unverifiedAuthor, setUnverifiedAuthor] = useState<UnverifiedContributor | null>(null);
   const orderedAuthors = contributors.map((contributor, index) => ({ ...contributor, sequence: index + 1 }));
 
+  const [width, setWidth] = useState(window.innerWidth);
```

Comment:
skal det ikke være height? 🤔

Reply:
nei, vanlig å sjekke på bredde når man skal sjekke for mobil vs desktop. Finnes ikke mui-breakpoints for høyde liksom

Reply:
ah, tenkte denne var for å sjekke høyden på skjermen, og vise "Legg til"-knappen øverst avhengig av det